src/client/index.js

```diff
@@ -18,14 +18,25 @@ export const registerSnapshot = require('./snapshotsManager').registerSnapshot;
 
 const TAG = 'PIXELS_CATCHER::APP::SNAPSHOT';
 
-type ConfigType = { baseUrl?: string };
+type ConfigType = {
+  baseUrl?: string,
+  /**
+   * Callback for react-native-navigation
+   * @param snapshot Current snapshot
+   */
+  rnnSetup?: (snapshot: SnapshotsContainer) => void,
+};
 
 export const runSnapshots = (appName: string, config: ConfigType = {}) => {
   log.i(TAG, `Run snapshots for ${appName}`);
   log.i(TAG, `Config is:\n ${JSON.stringify(config, null, 2)}`);
-  const { baseUrl } = config;
+  const { baseUrl, rnnSetup } = config;
   if (baseUrl) {
     network.setBaseUrl(baseUrl);
   }
-  AppRegistry.registerComponent(appName, () => SnapshotsContainer);
+  if (rnnSetup) {
```

Comment:
Hi @makarkotlov, thanks for the PR and adding support for `React Native Navigation`.  That's nice that it can also work with it (I work with React Navigation). The change looks good, but I'm thinking about different name for `rnnSetup`. Because as you've mentioned in readme there is a line:

> Navigation.registerComponent

also by default 

> AppRegistry.registerComponent

is used. So I'm thinking maybe `registerComponent` can be a good name. That name is aligned with `AppRegistry` and `Navigation`. It is also a bit more generic, so theoretically can be used not only for react native navigation, but let's say with Provider, etc.